core/src/test/java/org/apache/mesos/offer/DefaultOfferRequirementProviderTest.java

```diff
@@ -26,10 +27,13 @@
     private static final double MEM = 1024.0;
     private static final PlacementRuleGenerator ALLOW_ALL = new PassthroughGenerator(new PassthroughRule("test"));
     private DefaultOfferRequirementProvider defaultOfferRequirementProvider;
+    private EnvironmentVariables environmentVariables;
```

Comment:
Is this used somewhere or does setting env vars on it simulate a real environment for tests?